README.md

```diff
@@ -1,2 +1,71 @@
-# reviews-service
-Repo for the Airbnb reviews component
+# Air Tree n Tree Reviews Service
+
+---
+
+## API Spec
+
+`GET /api/reviews/:roomid/?review_group=[0 || selected pagination]`
+
+### Parameters
+
+| Params | Type |
+| --- | --- |
+| :roomId | `String` |
+| selected pagination | `Number` |
+
+### Response:
+
+| Field | Type |
+| ----- | ---- |
+| roomId | `Number` | 
+| reviews| [`Reviews[]`](#Response-Property:-Reviews)|
+| averageRating| [`AverageRating{}`](#Response-Property:-Rating)|
```

Comment:
I think you meant (#Response-Property:Average-Rating)

Or you could change the header on line 32